content/guides/models/track/log/distributed-training.md

```diff
@@ -7,26 +7,32 @@ menu:
 title: Log distributed training experiments
 ---
 
-In distributed training, models are trained using multiple GPUs in parallel. W&B supports two patterns to track distributed training experiments:
+During a distributed training experiment, you train a model using multiple machines or clients in parallel. W&B can help you track distributed training experiments. Based on your use case, use one of the following approaches to track distributed training experiments:
```

Comment:
```suggestion
During a distributed training experiment, you train a model using multiple machines or clients in parallel. W&B can help you track distributed training experiments. Based on your use case, track distributed training experiments using of the following approaches:
```
Trying to avoid "your use case, use"